browser/src/app.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+const Logger = require('blgr');
+const SPVNode = require('../../lib/node/spvnode');
```

Comment:
@boymanjor I am running into an error where `this.timer.unref` is undefined. It appears that `spvnode` uses `setInterval().unref` which is available in node env but not browser.

I have added a quick polyfill and now the SPVNode can run in browser and sync!

```

const _setInterval = setInterval;

function Timer(t) {
  this.t = t;
}

Timer.prototype.valueOf = function() {
  return this.t;
};

Timer.prototype.unref = function() {
  clearInterval(this.t);
};

window.setInterval = function(fn, timeout) {
  return new Timer(_setInterval(fn, timeout));
};
```